packages/patternfly-4/react-core/src/components/Tabs/Tabs.js

```diff
@@ -149,65 +161,78 @@ class Tabs extends React.Component {
+            )}
+          >
+            <Tab {...child.props}
+              ref={(node) => { this.child = node; }}
+              className={css(styles.tabsButton)}
+              onClick={event => this.handleTabClick(event, child.props.eventKey, child.props.tabContentRef)}
+              id={`pf-tab-${child.props.eventKey}-${child.props.id || this.id}`}
+              aria-controls={`pf-tab-section-${child.props.eventKey}-${child.props.id || this.id}`}>
+                {child.props.title}
+            </Tab>
+          </li>
+        ))}
+      </ul>
+      {!isSecondary && (
+        <button
+          className={css(styles.tabsScrollButton)}
+          variant="plain"
+          aria-label={rightScrollAriaLabel}
+          onClick={this.scrollRight}
+        >
+          <AngleRightIcon />
+        </button>
+      )}
+    </React.Fragment>;
+
     return (
       <React.Fragment>
-        <div
-          {...props}
-          className={css(
-            styles.tabs,
-            isFilled && styles.modifiers.fill,
-            isSecondary && styles.modifiers.tabsSecondary,
-            showLeftScrollButton && styles.modifiers.start,
-            showRightScrollButton && styles.modifiers.end,
-            highlightLeftScrollButton && styles.modifiers.startCurrent,
-            highlightRightScrollButton && styles.modifiers.endCurrent,
-            className
-          )}
-        >
-          {!isSecondary && (
-            <button
-              className={css(styles.tabsScrollButton)}
-              variant="plain"
-              aria-label={leftScrollAriaLabel}
-              onClick={this.scrollLeft}
-            >
-              <AngleLeftIcon />
-            </button>
-          )}
-          <ul className={css(styles.tabsList)} ref={this.tabList} onScroll={this.handleScrollButtons}>
-            {children.map((child, index) => (
-              <li
-                key={index}
-                className={css(
-                  styles.tabsItem,
-                  child.props.eventKey === activeKey && styles.modifiers.current,
-                  className
-                )}
-              >
-                <button
-                  className={css(styles.tabsButton)}
-                  onClick={event => this.handleTabClick(event, child.props.eventKey)}
-                  id={`pf-tab-${child.props.eventKey}-${child.props.id || this.id}`}
-                  aria-controls={`pf-tab-section-${child.props.eventKey}-${child.props.id || this.id}`}
-                >
-                  {child.props.title}
-                </button>
-              </li>
-            ))}
-          </ul>
-          {!isSecondary && (
-            <button
-              className={css(styles.tabsScrollButton)}
-              variant="plain"
-              aria-label={rightScrollAriaLabel}
-              onClick={this.scrollRight}
-            >
-              <AngleRightIcon />
-            </button>
-          )}
-        </div>
+        {label !== ''
```

Comment:
I don't think we want to use the label here to determine if this is <nav> or not.  I think the consumer should decide what they want to use based on their use case. I suggest we have an isNav attribute that defaults to false and switch that way.  If isNav is true we should require the label.  I would also name it aria-label instead.

Reply:
This was my suggestion. I was hoping to simplify the process for the consumer to define the tabs as a `<nav>` since an `aria-label` is necessary if they are inside a `<nav>` and also since `aria-label` is useless if they are inside a `<div>`.

Reply:
I just wonder if the API makes sense this way?  Would it be intuitive for the consumer  to know that if they supply a label, it would change the internal structure of the Tab.  Also this approach does not prevent the consumer from adding an aria-label when it is a <div>.  You would have to have the prop named aria-label rather than label.
@jschuler @dlabaj thought on the API?

Reply:
This is why I opened #1529. I think the best option is to make the Tab's property be named `aria-label` and not get propagated to `...props`. In the ES6 object destructuring rename it to `ariaLabel` via `'aria-label': ariaLabel` if some logic needs to take place with it in the rendering.

Reply:
Agree that we should aim for consistency with the apis.  I would have aria-label as a separate prop instead of keying off the label value.

Reply:
@tlabaj @rebeccaalpert Can we just move the nav & div element out of tabs and create 2 new tab components?  We could refactor tab.js to be 2 internal components (InternalTabs and TabContainer).

e.g.:

`
const TabsWithNav = ({ ariaLabel, wrapperClass, ...props }) => (<nav><InternalTab{...props}></nav><InternalTabContainer{...props}/>
// 
const Tabs = ({...props}) => (<div><InternalTabs {...props}/></div><InternalTabContainer{...props}/>

// What use to be component in Tabs.js is now InternalTabs, and the map function after that gets moved into a InternalTabContainer component?
`

Then you can keep the implementation props with each component and the users would know what they are using.

Reply:
Yes I prefer either the composition approach that @dlabaj  has above, or a boolean with the component composing it internally.    I think composition is clearer to the consumer.  Also, with composition it is more like PF# (bootstrap implementation), so consumers you are coming from PF3 might have an easier time upgrading.

Reply:
Can we go ahead and do this with composition @rebeccaalpert ?

Reply:
Yeah, no problem.